### ISSUE TITLE
Glitched Researching

### ISSUE THREAD

**Issue by ipatix** [IntelOrca/OpenRCT2#885]
I somehow manged to get the Researching broken in one of my save file. It says "Initial researching" and I can't select anything to research. However, I can set the money I want to spend on researching and it actually affects in my finances but doesn't do anything.

Here is my save file (I'm playing the scenario "Africa - Oasis"):
https://dl.dropboxusercontent.com/u/28573353/OpenRCT2/Saves/rctd.SV6

**Comment by IntelOrca** [comment]
I get the picture below, there is no park entrance... not sure how this is Africa - Oasis.

However in this game, there is no money which is why you can't set how much to spend on researching. In the original game this window would not normally be available. There is one roller coaster that can be researched (Mine Train) before the research is completed. Although this doesn't seem right seeing as there are plenty of objects loaded. Perhaps the invention list is corrupted?

![image](https://cloud.githubusercontent.com/assets/1482259/6424183/812b1dbe-beeb-11e4-9171-02d1dc7df370.png)

**Comment by ipatix** [comment]
Oh, I'm sorry for the inconvenience. I forgot that OpenRCT2 has a different save file directory and I picked the wrong/older save file.
I reuploaded the file and the link above should now be correct.

**Comment by IntelOrca** [comment]
@ipatix How do you know that you haven't just researched everything? The fact that it still allows you to set funding cost is probably a bug in the original game.

**Comment by ipatix** [comment]
No, the thing is that it actually costs me $100/200/400 each month. And I doubt that it'd say "Intial research" if I've finished researching everything. I don't know a lot about this scenario but I've literally got nothing since I started the scenario (I think I got 2 stalls and then it was done).

In the original game it will automatically grey out all research priorities if you're done on them and it will stop the research fundings once all categories are done.
